types.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package iavl;
+
+option go_package = ".;iavl";
+
+// VersionMetadata defines the metadata associated with a committed IAVL version.
+message VersionMetadata {
+  // tree version for the corresponding metadata
+  int64 version = 1;
+
+  // the UNIX timestamp of when the metadata was committed to disk
+  int64 committed = 2;
+
+  // the UNIX timestamp of when the metadata was updated
+  int64 updated = 3;
```

Comment:
Why call this `updated` when it is only different from `committed` IF the version gets deleted? This will always be true since VersionedTree is immutable.

Think it makes more sense to just call this `deleted` and only set it on deletion

Reply:
No, actually the `Updated` timestamp will also be updated during manual flushing as well. See the #239 PR.